### DISCUSSION TITLE
config file

### DISCUSSION THREAD

**Discussion by recursos** [streamlink/streamlink#3679] [Q&A] [answered]
This command line works:
streamlink --http-header  "User-Agent=xxxxx" --http-header "Referer=xxxx" "https://ondemand-streaming.rtp.pt/nas2.share/h264/512x384/p8677/p8677_1_20210406161005e005t1633d/master.m3u8" best --output "EP5-Prova de Vida-2021-04-06.mp4"

Tried to convert it to config file:

http-header=User-Agent=xxxxx
http-header=Referer=xxxx
url="https://ondemand-streaming.rtp.pt/nas2.share/h264/512x384/p8677/p8677_1_20210406161005e005t1633d/master.m3u8"
default-stream=best
output="EP5-Prova de Vida-2021-04-06-S.mp4"

but streamlink fails:
error: No plugin can handle URL: "https://ondemand-streaming.rtp.pt/nas2.share/h264/512x384/p8677/p8677_1_20210406161005e005t1633d/master.m3u8"

What is wrong with --url option?

Thanks,

**Comment by bastimeyer** [comment, marked as answer]
You've put the URL in the config file in quotes, which means it's part of the argument value, and this doesn't make sense.
Quotes on the command line on the other hand get interpreted by the shell when tokenizing the input and are thus not part of the argument value unless you've escaped those characters.

**Comment by recursos** [comment]
That solved it, thank you.
